docs/topics/kapt.md

```diff
@@ -98,6 +100,29 @@ To use the Gradle worker API for parallel execution of kapt tasks, add this line
 kapt.use.worker.api=true
 ```
 
+### Caching for annotation processors' classloaders (since 1.5.20)
+
+> Caching for annotation processors' classloaders in kapt is [Experimental](components-stability.md).
+> It may be dropped or changed at any time. Use it only for evaluation purposes.
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-28901).
+>
+{type="warning"}
+
+This experimental feature makes it possible to cache the classloaders of annotation processors in kapt and so 
+increases the speed of kapt for many consecutive Gradle runs.
+
+To enable this feature, use the following properties in your `gradle.properties` file:
+
+```groovy
+#positive value will enable caching
```

Comment:
I absolutely agree. It seemed to me that I saw somewhere the usage without spaces. But maybe I mixed up sth because I didn't find any examples without whitespaces. Will fix.